src/pages/solutions/spatial-data-management.astro

```diff
@@ -111,13 +115,13 @@ const page: IStandardPageMetadata = {
       class="w-full max-w-[400px] rounded-2xl"
     />
     <p>
-      Once your data sets are built, accessible, and current, consider contributing them to the State Geographic
-      Information Datasource (SGID) for maximal exposure and usage. The purpose of the SGID is to make the most relevant
-      and authoritative Utah spatial data available to the greatest audience possible. In some cases, UGRC will host
-      data for you, yet the preference is to enable agencies to be informed and responsible data stewards so we can
-      highlight your curated services, particularly through your ArcGIS Enterprise Portal, or ArcGIS Online. We can then
-      share your data directly with other users of our own ArcGIS Hub website to make your work easier for other data
-      consumers to find.
+      Once your datasets are built, accessible, and current, consider contributing them to the State Geographic
+      Information Datasource (SGID) for maximal exposure and usage. The SGID's goal is to make the most relevant and
+      authoritative Utah spatial data available to the largest possible audience. Ideally, we can help people find the
+      data you've already published by adding it to the <a href="/products/sgid/sgid-index">SGID Index</a> or the <a
+        href="https://opendata.gis.utah.gov">SGID on ArcGIS</a
```

Comment:
**issue**: this needs the external link tag

Reply:
I've been thinking about that- do we need external link tags for stuff pointing to other `.utah.gov` addresses? or do we have to tag everything going outside of `gis.utah.gov`? To me, the external link tag indicates its something out of our control and not an official state site.

Reply:
anything outside of our agency website so anything not hosted on `https://gis.utah.gov`